common/traits/000_wc_leadership_traits.txt

```diff
@@ -0,0 +1,195 @@
+# Written by: DukeJeans
+# Created: 1/22/2019
+# Modified: 3/28/2019
+#Leadership traits
+
+#Warcraft
+archmagus_leader = {
+	leader = yes
+	random = no
+	customizer = no
+
+	command_modifier = {
+		light_infantry = 0.2
+	}
+}
+
+light_foot_leader = {
+	leader = yes
+
+	command_modifier = {
+		heavy_infantry = 0.2
+	}
+}
+
+cavalry_leader = {
+	leader = yes
+
+	command_modifier = {
+		light_cavalry = 0.2
+	}
+}
+
+inspiring_leader = {
+	leader = yes
+
+	command_modifier = {
+		morale_offence = 0.15
+		morale_defence = 0.15
+	}
+}
+
+trickster = {
+	leader = yes
+
+	command_modifier = {
+		random = 0.3
+	}
+}
+
+organizer = {
+	leader = yes
+
+	command_modifier = {
+		speed = 0.2
+		retreat = 0.1
+	}
+}
+
+defensive_leader = {
+	leader = yes
+
+	command_modifier = {
+		defence = 0.25
+		damage = -0.1
+	}
+}
+
+# Renamed to "Direct Leader"
+experimenter = {
+	leader = yes
+
+	command_modifier = {
+		center = 0.2
+	}
+}
+
+flanker = {
+	leader = yes
+
+	command_modifier = {
+		flank = 0.2
+	}
+}
+
+aggressive_leader = {
+	leader = yes
+
+	command_modifier = {
+		pursue = 0.25
+		damage = 0.1
+		defence = -0.1
+	}
+}
+
+siege_leader = {
+	leader = yes
+
+	command_modifier = {
+		siege = 0.4
+	}
+}
+
+marksman_captain_leader = {
+    leader = yes
+
+    command_modifier = {
+        archers = 0.20
+    }
+}
+
+aerial_raider_leader = {
+    leader = yes
+
+    command_modifier = {
+        pikemen = 0.20
+    }
+}
+
+heavy_infantry_leader = {
+    leader = yes
+
+    command_modifier = {
+        knights = 0.20
+    }
+}
+
+taskmaster_leader = {
+    leader = yes
+	random = no
+
+    command_modifier = {
+        thralls = 0.20
+		speed = 0.10
+		supply_limit = 3
+    }
+}
+
+artillery_engineer_leader = {
+    leader = yes
+
+    command_modifier = {
+        siege_machines = 0.20
+        war_machines = 0.20
+    }
+
+	siege_speed = 0.15
+}
+
+warpriest_leader = {
+    leader = yes
+	random = no
+	customizer = no
+
+    command_modifier = {
+        morale_defence = 0.30
+        max_attrition = -0.50
+        days_of_supply = 30
+    }
+}
+
+offensive_caster_leader = {
+    leader = yes
+	random = no
+	customizer = no
+
+    command_modifier = {
+		light_infantry = 0.20
+        damage = 0.05
+		pursue = 0.10
+    }
+}
+
+defensive_caster_leader = {
+    leader = yes
+	random = no
+	customizer = no
+
+	command_modifier = {
+		light_infantry = 0.20
+        defence = 0.05
+		retreat = 0.10
+    }
+}
+
+defiler_leader = {
+    leader = yes
+	trait_effect_capture_commanders = 1
```

Comment:
(not really)

Reply:
It is just a text, not an effect.
I did not find any events that would use this trait to capture commanders.

Reply:
Interesting, I took it from the Norse lodge commander trait, I'll have to dig around vanilla